content/sensu-go/5.0/getting-started/sample-app.md

```diff
@@ -0,0 +1,380 @@
+Let’s say we want to receive a Slack alert if the dummy app returns an unhealthy response.
+We can create a Sensu pipeline to send alerts to a #demo channel in Slack using the [Sensu Slack plugin](https://github.com/sensu/sensu-slack-handler).
+Sensu Plugins are open-source collections of Sensu building blocks shared by the Sensu Community.
+
+**1. Create an asset to help agents find and install the [Sensu Slack handler](https://github.com/sensu/sensu-slack-handler)**
+
+{{< highlight shell >}}
+sensuctl create -f go/config/assets/slack-handler.yaml
+{{< /highlight >}}
+
+**2. Get your Slack webhook URL and add it to `go/config/handlers/slack.yaml`.**
+
+If you're already an admin of a Slack, visit `https://YOUR WORKSPACE NAME HERE.slack.com/services/new/incoming-webhook` and follow the steps to add the Incoming WebHooks integration and save the settings.
+(If you're not yet a Slack admin, start [here](https://slack.com/get-started#create) to create a new workspace.)
+After saving, you'll see your webhook URL under Integration Settings.
+
+Open `go/config/handlers/slack.yaml` and replace `SECRET` in the following line with your Slack workspace webhook URL:
+
+{{< highlight shell >}}
+"command": "slack-handler --channel '#demo' --timeout 20 --username 'sensu' --webhook-url 'SECRET'",
+{{< /highlight >}}
+
+So it looks something like:
+
+{{< highlight shell >}}
+"command": "slack-handler --channel '#demo' --timeout 20 --username 'sensu' --webhook-url 'https://hooks.slack.com/services/XXXXXXXXXXXXXXXX'",
+{{< /highlight >}}
+
+**3. Create a handler to send alert events to Slack that uses the `slack-handler` asset.**
+
+{{< highlight shell >}}
+sensuctl create -f go/config/handlers/slack.yaml
+{{< /highlight >}}
+
+### Create a Sensu service check to monitor the status of the dummy app
+
+To automatically monitor the status of the dummy app, we'll create an asset that lets the Sensu agents use a [Sensu HTTP plugin](https://github.com/portertech/sensu-plugins-go).
+
+**1. Create the `check-plugins` asset.**
+
+{{< highlight shell >}}
+sensuctl create -f go/config/assets/check-plugins.yaml
+{{< /highlight >}}
+
+**2. Now we can create a check to monitor the status of the dummy app that uses the `check-plugins` asset and the Slack pipeline.**
+
+{{< highlight shell >}}
+sensuctl create -f go/config/checks/dummy-app-healthz.yaml
+{{< /highlight >}}
+
+**3. With the automated alert workflow in place, we can see the resulting events in the Sensu dashboard.**
+
+At http://webui.sensu.local/signin, sign in with your sensuctl username (`demo`) and password.
+
+**4. Toggle the health of the dummy app to simulate a failure.**
+
+{{< highlight shell >}}
+curl -iXPOST http://dummy.local/healthz
+{{< /highlight >}}
+
+We should now be able to see a critical alert in the [Sensu dashboard](http://webui.sensu.local/events) as well as by using sensuctl:
+
+{{< highlight shell >}}
+sensuctl event list
+{{< /highlight >}}
+
+You should also see an alert in Slack.
```

Comment:
This check needs the `is_incident` filter

Reply:
Already present https://github.com/sensu/sensu-kube-demo/blob/master/go/config/handlers/slack.yaml#L13